packages/vue/src/components/molecules/SfComponentSelect/SfComponentSelect.stories.js

```diff
@@ -242,6 +242,19 @@ export default {
       defaultValue: "",
       description: "The content of the option",
     },
+    cancelLabel: {
+      control: "text",
+      table: {
+        category: "Props for main component",
+        type: {
+          summary: "string",
+        },
+        defaultValue: {
+          summary: "Cancel",
+        },
+      },
+      description: "Cancel label text",
```

Comment:
It's nice to have defaultValue, also set here, because it will display field with default value in docs instead of this `set string` button 
![Screenshot from 2021-11-18 09-22-01](https://user-images.githubusercontent.com/32042425/142378595-6d640b2a-351a-4850-8a75-575a8cc1bfbd.png)